spec.bs

```diff
@@ -2027,6 +2115,27 @@ The network revocation mechanism requires the following monkeypatches to the [[F
 
   Add "[=must be blocked due to a revoked partition nonce=]" to the conditions after
   "should request be blocked by Content Security Policy".
+
+  Issue: This needs to be passed in both the fenced frame nonce as well as the iframe credentialless
+  nonce, if it exists.
+  (<a href="https://github.com/WICG/fenced-frame/issues/191">WICG/fenced-frame#191</a>)
+</div>
+
+The network revocation mechanism requires the following monkeypatches to the [[HTML]] Standard.
+
+<div algorithm=network-status-document-abort>
+  Add a new step to the [=abort a document=] algorithm after step 4 that says:
+
+  5. [=Recalculate the untrusted network status of all fenced frame descendants=] given <var
+     ignore>document</var>'s [=node navigable=]'s [=navigable/top-level traversable=].
+</div>
+
+<div algorithm=network-status-navigate>
+  Modify the call to [=attempt to populate the history entry's document=] in the [=navigate=]
+  algorithm. Update the [=completionSteps=] to include the following new step:
+
+  2. [=Recalculate the untrusted network status of all fenced frame descendants=] given <var
```

Comment:
Same parallel issue exists here as above, I think. Maybe we want to, on the same line, conditionally go in parallel if we're not already in parallel.

However I still have another question: why do we have to recalculate after this specific milestone?

Reply:
Right now, this won't do anything. This is really only going to be useful when https://github.com/WICG/fenced-frame/issues/173 is added to the spec. For now, I'm going to take this out, and it can be added back in once the stop in-progress navigation work is added.